### PR TITLE
Con 785 fix data refresh

### PR DESCRIPTION
**Pull request by cofaulco**

### CODE REVIEW

File: src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs [hunk 1]
```diff
@@ -32,22 +32,22 @@ public async Task<EmployerSurveyInvite> GetEmployerInviteForUniqueCode(Guid uniq
 
         public async Task<IEnumerable<EmployerSurveyInvite>> GetEmployerUsersToBeSentInvite()
         {
-            return await _dbConnection.QueryAsync<EmployerSurveyInvite>(sql: $@"
-                                        SELECT * 
-                                        FROM {EmployerSurveyInvites}
```
Comment (lazcool) [inline src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs]: `EmployerSurveyInvites` can be removed now.

File: src/Employer/ESFA.DAS.ProvideFeedback.Emailer/SurveyInviteGenerator.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+namespace ESFA.DAS.Feedback.Employer.Emailer
+{
+    public class SurveyInviteGenerator
+    {
+        private readonly EmailSettings _emailSettingsConfig;
+        private readonly IStoreEmployerEmailDetails _employerEmailDetailRepository;
+        private readonly ILogger<SurveyInviteGenerator> _logger;
+
+        public SurveyInviteGenerator(
+            IOptions<EmailSettings> options,
```
Comment (lazcool) [inline src/Employer/ESFA.DAS.ProvideFeedback.Emailer/SurveyInviteGenerator.cs]: Microsoft seems to be [suggesting ](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-2.2#options-interfaces)to use `IOptionsMonitor` in new code.

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/EmployerDataRetreiverFunction.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        public EmployerDataRetreiverFunction(
+            EmployerFeedbackDataRetrievalService dataRetrievalService,
+            ILogger<EmployerDataRetreiverFunction> logger)
+        {
+            _dataRetrievalService = dataRetrievalService;
+            _logger = logger;
+        }
+
+        [FunctionName("EmployerDataRetreiverFunction")]
```
Comment (lazcool) [inline src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/EmployerDataRetreiverFunction.cs]: Spelling.

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/InitiateDataRefreshFunction.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+        [FunctionName("EmployerFeedbackDataRefreshFunction")]
+        [return: ServiceBus("%RetrieveFeedbackDataMessageQueueName%", Connection = "ServiceBusConnection")]
+        public async Task<string> Run([TimerTrigger("%DataRefreshSchedule%", RunOnStartup = true)]TimerInfo myTimer, ILogger log)
+        {
+            _logger.LogInformation("Starting invite data refresh.");
+            try
+            {
+                await _dbRepository.ResetFeedback();
+                return string.Empty;
```
Comment (lazcool) [inline src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/InitiateDataRefreshFunction.cs]: What's the purpose of returning a string that's always empty?

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyInviteEmailer.cs [hunk 5]
```diff
@@ -17,7 +17,7 @@ public SurveyInviteEmailer(EmployerSurveyInviteEmailer inviteEmailer)
 
         [FunctionName("SurveyInviteEmailer")]
         public async Task Run(
-            [TimerTrigger("%InviteEmailerSchedule%")]TimerInfo myTimer,
+            //[TimerTrigger("%InviteEmailerSchedule%")]TimerInfo myTimer,
```
Comment (lazcool) [inline src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyInviteEmailer.cs]: Is this meant to be checked-in?

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyReminderEmailer.cs [hunk 6]
```diff
@@ -17,7 +17,7 @@ public SurveyReminderEmailer(EmployerSurveyReminderEmailer reminderEmailer)
 
         [FunctionName("SurveyReminderEmailer")]
         public async Task Run(
-            [TimerTrigger("%ReminderEmailerSchedule%")]TimerInfo myTimer,
+            //[TimerTrigger("%ReminderEmailerSchedule%")]TimerInfo myTimer,
```
Comment (lazcool) [inline src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyReminderEmailer.cs]: Same here.

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/EmployerDataRetreiverFunction.cs (hunk 3 above)
Comment (cofaulco) [reply src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/EmployerDataRetreiverFunction.cs]: 'i before e' and all that

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/InitiateDataRefreshFunction.cs (hunk 4 above)
Comment (cofaulco) [reply src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/InitiateDataRefreshFunction.cs]: puts an empty message on the queue to start the processing

File: src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs (hunk 1 above)
Comment (cofaulco) [reply src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/EmployerFeedbackRepository.cs]: removed

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyInviteEmailer.cs (hunk 5 above)
Comment (cofaulco) [reply src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyInviteEmailer.cs]: nope
fixed

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/EmployerDataRetreiverFunction.cs (hunk 3 above)
Comment (cofaulco) [reply src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/EmployerDataRetreiverFunction.cs]: fixed

File: src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyReminderEmailer.cs (hunk 6 above)
Comment (cofaulco) [reply src/Employer/ESFA.DAS.ProvideFeedback.Employer.Functions.Emailer/SurveyReminderEmailer.cs]: fixed

**Pull request merged**